forc/src/utils/helpers.rs

```diff
@@ -76,6 +76,13 @@ pub fn default_output_directory(manifest_dir: &Path) -> PathBuf {
     manifest_dir.join(DEFAULT_OUTPUT_DIRECTORY)
 }
 
+/// Returns the user's `.forc` directory, `$HOME/.forc` by default.
+pub fn user_forc_directory() -> PathBuf {
+    dirs::home_dir()
+        .expect("unable to find the user home directory")
+        .join(".forc")
```

Comment:
Should use `FORC_DEPENDENCIES_DIRECTORY` here. Maybe rename the constant to something more general.

https://github.com/FuelLabs/sway/blob/be7b2d48b0afe6427bf81482e663ae17773fe6d9/sway-utils/src/constants.rs#L5

Reply:
Nice, renaming this helped to find a few more spots that could be refactored in terms of the new function too :+1: